.gitignore

```diff
@@ -24,3 +24,4 @@ build/
 
 # CocoaPods
 /ios/Pods/
+android/app/google-services.json
```

Comment:
I think we should add a comment here so that it does not feel merged to `CocoaPods` block 🤓